build.sbt

```diff
@@ -2592,6 +2599,7 @@ lazy val `std-base` = project
       `base-polyglot-root` / "std-base.jar",
     libraryDependencies ++= Seq(
       "org.graalvm.polyglot" % "polyglot"                % graalMavenPackagesVersion,
+      "org.graalvm.sdk"      % "collections"             % graalMavenPackagesVersion,
```

Comment:
Collections are now separated into its own JAR. As the group ID suggests (`org.graalvm.sdk`) they are not part of the `org.graalvm.polyglot` ecosystem. We should probably **stop using them** and _not expose their classes_ to libraries. CCing @radeusgd

Reply:
As of 253ded5 I am able to execute:
```
enso$ sbt 
sbt> project runtime
sbt:runtime> testOnly *ForeignMethodInvokeTest -- -z org.enso.interpreter.test.ForeignMethodInvokeTest.testParallelInteropWithJavaScript
```
and that is enough for testing https://github.com/oracle/graal/pull/8266